dbms/tests/queries/0_stateless/00700_decimal_empty_aggregates.sql

```diff
@@ -0,0 +1,90 @@
+DROP TABLE IF EXISTS decimal;
+
+CREATE TABLE decimal
+(
+    a Decimal32(4),
+    b Decimal64(8),
+    c Decimal128(8)
+) ENGINE = Memory;
+
+SELECT count(a), count(b), count(c) FROM decimal;
+SELECT [min(a), max(a)], [min(b), max(b)], [min(c), max(c)] FROM decimal;
+
+SELECT sum(a), sum(b), sum(c), sumWithOverflow(a), sumWithOverflow(b), sumWithOverflow(c) FROM decimal;
+SELECT sum(a), sum(b), sum(c), sumWithOverflow(a), sumWithOverflow(b), sumWithOverflow(c) FROM decimal WHERE a > 0;
+SELECT sum(a), sum(b), sum(c), sumWithOverflow(a), sumWithOverflow(b), sumWithOverflow(c) FROM decimal WHERE a < 0;
+SELECT sum(a+1), sum(b+1), sum(c+1), sumWithOverflow(a+1), sumWithOverflow(b+1), sumWithOverflow(c+1) FROM decimal;
+SELECT sum(a-1), sum(b-1), sum(c-1), sumWithOverflow(a-1), sumWithOverflow(b-1), sumWithOverflow(c-1) FROM decimal;
+
+SELECT avg(a), avg(b), avg(c) FROM decimal;
+SELECT avg(a), avg(b), avg(c) FROM decimal WHERE a > 0;
+SELECT avg(a), avg(b), avg(c) FROM decimal WHERE a < 0;
+
+SELECT (uniq(a), uniq(b), uniq(c)),
+    (uniqCombined(a), uniqCombined(b), uniqCombined(c)),
+    (uniqCombined(17)(a), uniqCombined(17)(b), uniqCombined(17)(c)),
+    (uniqExact(a), uniqExact(b), uniqExact(c)),
+    (uniqHLL12(a), uniqHLL12(b), uniqHLL12(c))
+FROM (SELECT * FROM decimal ORDER BY a);
+
+SELECT uniqUpTo(10)(a), uniqUpTo(10)(b), uniqUpTo(10)(c) FROM decimal WHERE a >= 0 AND a < 5;
+SELECT uniqUpTo(10)(a), uniqUpTo(10)(b), uniqUpTo(10)(c) FROM decimal WHERE a >= 0 AND a < 10;
+
+SELECT argMin(a, b), argMin(a, c), argMin(b, a), argMin(b, c), argMin(c, a), argMin(c, b) FROM decimal;
+SELECT argMin(a, b), argMin(a, c), argMin(b, a), argMin(b, c), argMin(c, a), argMin(c, b) FROM decimal WHERE a > 0;
+SELECT argMax(a, b), argMax(a, c), argMax(b, a), argMax(b, c), argMax(c, a), argMax(c, b) FROM decimal;
+SELECT argMax(a, b), argMax(a, c), argMax(b, a), argMax(b, c), argMax(c, a), argMax(c, b) FROM decimal WHERE a < 0;
+
+SELECT median(a), median(b), median(c) as x, toTypeName(x) FROM decimal;
```

Comment:
We discussed that toTypeName(a) and toTypeName(b) should return Decimal(9, 4) and Decimal(18, 8)

Reply:
I changed types in testing table for empty test.
now it like this:
```
    a Decimal(6, 4),
    b Decimal(16, 7),
    c Decimal(20, 8)
```

Reply:
for quantile and median added test for result types